spec/paratrooper/source_control_spec.rb

```diff
@@ -156,7 +169,7 @@
     context "when branch_name is a string" do
       it 'pushes branch_name' do
         config = instance_double(Paratrooper::Configuration, force_push: false,
-          deployment_host: "HOST", app_name: "APP", branch_name?: true,
+          deployment_host: "HOST", app_name: "APP", remote_name: "APP", branch_name?: true,
           branch_name: "BRANCH_NAME", tag_name: nil,
           system_caller: system_caller
         )
```

Comment:
Is there a need to add `remote_name` to these existing scenarios?

Reply:
RSpec creates a "pure double". Because of that the call here https://github.com/mattpolito/paratrooper/pull/87/commits/f8d5578b9b56472e46c68d8be58800239b477964#diff-4df900f9b9579687af92db8b5b6ed30bR98 always failed, because the behaviour of the `remote_name` attribute reader wasn't defined. First I worked around with some `expect(config).to receive(:remote_name).and_return('APP')` construct. I'm not that familiar with RSpec in that regard, so I failed to do it _the right way_ I guess. Most likely for such cases the object should not be a pure double (created via `instance_double` here https://github.com/mattpolito/paratrooper/pull/87/commits/f8d5578b9b56472e46c68d8be58800239b477964#diff-3359d3cb4708a538a0312752ea92d294R8)?